packages/react/src/tests/formatConversion/exportParseEquality/testExportParseEquality.test.ts

```diff
@@ -0,0 +1,19 @@
+import { testExportParseEquality } from "@blocknote/core";
+import { describe, it } from "vitest";
+
+import { setupTestEditor } from "../../setupTestEditor.js";
+import { testSchema } from "../../testSchema.js";
+import { getExportParseEqualityTestCases } from "./getExportParseEqualityTestCases.js";
+
+describe("Export/parse equality tests", () => {
+  const getEditor = setupTestEditor(testSchema);
+
+  for (const testCase of [
+    ...getExportParseEqualityTestCases("blocknoteHTML"),
+    ...getExportParseEqualityTestCases("nodes"),
+  ]) {
```

Comment:
Agree with Yousef, these should be their own describe blocks which help isolate tests to groups that are relevant to what you are doing